.github/workflows/update-flake.yml

```diff
@@ -0,0 +1,23 @@
+name: Update flake inputs
+on:
+  schedule:
+    # Update every Sunday and Wednesday
+    - cron: "51 3 * * 0,3"
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Install Nix
+        uses: cachix/install-nix-action@v17
+        with:
+          extra_nix_config: |
+            access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
```

Comment:
We probably don't need this.

Reply:
You're probably right, since this is just an update job. It wouldn't hurt to have though, right? I noticed nix-community/infra has this [in their update workflow](https://github.com/nix-community/infra/blob/fa549f5e591a41b317978023bad7b2ea91503e19/.github/workflows/flake-updates.yml#L17).

Reply:
I'd rather avoid giving unnecessary privileges to things, and I suspect people are cargo culting this from the update-flake-lock [README](https://github.com/DeterminateSystems/update-flake-lock#example).

Reply:
That's fair. I have nothing against removing it, so I'll just take it out.